autometa/common/external/prodigal.py

```diff
@@ -185,7 +185,7 @@ def run(assembly, nucls_out, prots_out, force=False, cpus=0, parallel=True):
             with open(fp) as fh:
                 for _ in SimpleFastaParser(fh):
                     pass
-        except (IOError, ValueError):
+        except (OSError, ValueError):
```

Comment:
Does OSError result from `SimpleFastaParser`?

Reply:
I was unable to replicate the error. But I must have seen it being raised instead of `IOError` and that's why I would have put it there. Moreover, from [this](https://stackoverflow.com/a/29347946/12671809) discussion it seems we should be using `OSError` instead of `IOError` as `IOError` is now just an alias for `OSError`

Reply:
Unfortunately BioPython has not specified the errors its packages are raising in its [documentation](https://biopython.org/docs/1.75/api/Bio.SeqIO.FastaIO.html#Bio.SeqIO.FastaIO.SimpleFastaParser).
If you want we can catch both `IOError1 and `OSError`

Reply:
We should probably raise `OSError` rather than `IOError` as well.